src/components/ansatte/Ansatte.tsx

```diff
@@ -53,6 +53,9 @@ const Ansatte = () => {
                 Har du ansatte som har vært borte i to eller flere ikke-sammenhengende perioder
                 <Link to="../enkel/"> skal du bruke et eget skjema som du finner her.</Link>
               </Normaltekst>
+              <Normaltekst>
+                Har dere svært mange ansatte kan det om ønskelig <Link to="../excel/">benyttes Excel-opplasting.</Link>
```

Comment:
Kanskje bruke Lenke-komponenten her også?

Reply:
Var det noen som sa "practise what you preach"? :-)